README.md

```diff
@@ -0,0 +1,24 @@
+# ☁ Clowder Backend
+
+## Prerequisites  
+This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance
+
+## Setup
+### Setting up the database
+
+1. Ensure that Docker is started. This is down to how you have installed Docker. You can test Docker is online by running `docker info`
+2. Run `./setup-db.sh local up -d`.
```

Comment:
1. This should be added as a script in package.json
2. When I run this command I get the error:

```
✘ postgres Error failed to resolve reference "docker.io/library/postgres:14": failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://auth.docker.io/token?s...                 1.1s 
Error response from daemon: failed to resolve reference "docker.io/library/postgres:14": failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://auth.docker.io/token?scope=repository%3Alibrary%2Fpostgres%3Apull&service=registry.docker.io: 401 Unauthorized
```

Reply:
That error seems to be an issue with your local Docker configuration, nothing I can fix in this codebase. What script would you want to put in the package.json?